ocaml/libs/xapi-stdext/lib/xapi-stdext-std/listext.mli

```diff
@@ -27,6 +27,8 @@ module List : sig
   (** [drop n list] returns the list without the first [n] elements of [list]
       (or [] if list is shorter). *)
 
+  val last : 'a list -> 'a
```

Comment:
Add some documentation. "Return the last element of a list or raise Invalid_argument if the list is empty"